xml/deployment_additional.xml

```diff
@@ -68,36 +68,73 @@ if necessary.
   </important>
 
   <para>
-   To install the &kube; dashboard, perform the following steps:
+   Install the &kube; dashboard. 
   </para>
 
+  <note>
+   <title>Requirements</title>
+    <para>
+      Heapster version 1.3.0 or later must be installed to the cluster.  
+      helm instructions to install the upstream version are included in this section. 
+    </para>
+  </note>
+
+  <note>
+   <title>Tool versions</title>
+    <para>
+      helm version 2.7.2+ and kubectl version 1.8.0+ recommended
+    </para>
+  </note>
+
   <procedure>
    <title>Installation of &kube; Dashboard</title>
+   <step>
+     <screen>
+helm install --name heapster-default --namespace=kube-system stable/heapster --version=0.2.7 --set rbac.create=true
+     </screen>
+   </step>
+   <step>
+     <screen>
+helm install --namespace=kube-system --name=kubernetes-dashboard stable/kubernetes-dashboard --version=0.6.1
+     </screen>
+   </step>
    <step>
     <para>
-     Connect to the &admin_node; using SSH.
+      SSH into any cluster node. 
     </para>
    </step>
    <step>
     <para>
-     Run the command:
+      Visit <literal>http://127.0.0.1:8081/</literal> in your browser. 
+      You should be greeted with a Welcome page. There is one Configure dialog with 2 options, <literal>kubeconfig</literal> and <literal>token</literal> authentication. 
     </para>
-<screen>kubectl create -f https://raw.githubusercontent.com/kubernetes/dashboard/v1.6.3/src/deploy/kubernetes-dashboard.yaml</screen>
    </step>
    <step>
     <para>
-     After the previous command finishes, run the following to expose the
-     proxy:
+      Select <literal>token</literal> authentication. To retrieve your token refer to the value in your kubeconfig file by running the command:
```

Comment:
The kubeconfig is the file the user downloads from Velum after bootstrap. It can be placed anywhere on the host system so I did not want to put in an explicit path.